network-api/networkapi/wagtailpages/pagemodels/customblocks/accordion_block.py

```diff
@@ -0,0 +1,37 @@
+from wagtail.core import blocks
+from ..customblocks.base_rich_text_options import base_rich_text_options
+from .datawrapper_block import DatawrapperBlock
+from .image_block import ImageBlock
+
+accordion_rich_text = blocks.RichTextBlock(
+    features=base_rich_text_options + ['ul', 'ol', 'document-link'],
+    blank=True
+)
+
+
+class AccordionItem(blocks.StructBlock):
```

Comment:
Hey @fessehaye heres my suggestion for this PR, which is moving the richtext/datawrapper/image streamblocks into a new StructBlock, that way in the CMS we can keep all accordion items under one accordion component.